docs/source/data_entry.rst

```diff
@@ -0,0 +1,85 @@
+.. data_entry:
+
+Data Entry
+=========================
+
+Creating and Data Model Objects
+---------------------------------
+
+Each data object and template in the GEMD_ data model has a corresponding resource in the Citrine Python Client.
+For example, the :class:`citrine.resources.process_spec.ProcessSpec` class implements the ProcessSpec_ object in GEMD_.
+The Citrine Python Client implements are consistent with the GEMD_ model specification.
+
+The Citrine Python Client is built on top of and entirely interoperable with the taurus_ package.
+Any method that accepts the Citrine Python Client's implementations of data model objects should also accept those from taurus.
+
+Identifiying Data Model Objects
+---------------------------------
+
+After registering a data model object, you will probably want to be able to find that object again.
+The easiest way to get an existing object is by one of its unique identifiers.
+Every data model object on the Citrine Platform has a platform-issued identifier, often referred to as the "Citrine Identifier" or "CitrineId".
+These identifiers are UUID4_, which are extremely robust but also not especially human readable.
+
+`Alternative identifiers`__ are an even easier way to recall data objects.
+To create an alternative identifier, simply add key-value pairs to the ``uids`` dictionary in the data model object.
+The key defines the ``scope`` and the value the ``id``.
+As a pair, they must be unique across the entire platform.
+You can think of each value of ``scope`` as defining a namespace, with ``id`` being the name within that namespace.
+
+__ https://citrineinformatics.github.io/gemd-docs/specification/unique-identifiers/#alternative-ids
+
+Registering Data Model Objects
+---------------------------------
+
+Data model objects are created on the Citrine Platform through the ``register`` method present in each data model object collection.
+For example:
+
+.. code-block:: python
+
+    project.process_specs.register(ProcessSpec(...))
+
+If you have taurus_ objects, e.g. :class:`taurus.entity.object.process_spec.ProcessSpec`, you can register it just like the objects defined in the Citrine Python Client.
```

Comment:
Change taurus to GEMD?

Reply:
@djack : how imminent is the taurus -> GEMD rename?